module5/src/com/epam/dragon/util/TreasureAggregator.java

```diff
@@ -0,0 +1,110 @@
+        treasures[54] = new Treasure("The serrated dagger", 865);
+        treasures[55] = new Treasure("Shortened dagger", 653);
+        treasures[56] = new Treasure("Sharpening", 53);
+        treasures[57] = new Treasure("Blunt", 67);
+        treasures[58] = new Treasure("Combat crowbar", 678);
+        treasures[59] = new Treasure("Double-sided cleaver", 553);
+        treasures[60] = new Treasure("Nutcracker", 532);
+        treasures[61] = new Treasure("Jace's Hammer of Vigilance", 578);
+        treasures[62] = new Treasure("The royal sword of swiftness", 554);
+        treasures[63] = new Treasure("Griswold's serrated sword", 335);
+        treasures[64] = new Treasure("The blade of the wurdalak King", 6789);
+        treasures[65] = new Treasure("Assassin of the Gods", 8765);
+        treasures[66] = new Treasure("Amber Wing", 345);
+        treasures[67] = new Treasure("Saber", 643);
+        treasures[68] = new Treasure("Steel Veil", 735);
+        treasures[69] = new Treasure("Leorik's Crown", 733);
+        treasures[70] = new Treasure("Jester's comb", 6532);
+        treasures[71] = new Treasure("Crown of the undead", 542);
+        treasures[72] = new Treasure("The mysterious helmet", 247);
+        treasures[73] = new Treasure("Hood", 673);
+        treasures[74] = new Treasure("Ringed hood", 321);
+        treasures[75] = new Treasure("Mysterious shoulder pads", 457);
+        treasures[76] = new Treasure("Leather robe", 432);
+        treasures[77] = new Treasure("Shoulder of the Skeleton King", 765);
+        treasures[78] = new Treasure("The will of Razet.", 431);
+        treasures[79] = new Treasure("Hellfire amulet", 368);
+        treasures[80] = new Treasure("Optical amulet", 753);
+        treasures[81] = new Treasure("The mysterious amulet", 428);
+        treasures[82] = new Treasure("Hoppy necklace", 876);
+        treasures[83] = new Treasure("The patronage of the moonlight", 985);
+        treasures[84] = new Treasure("Leorik's golden necklace", 564);
+        treasures[85] = new Treasure("Kalan's Wisdom", 367);
+        treasures[86] = new Treasure("Mirror", 732);
+        treasures[87] = new Treasure("The Eye", 789);
+        treasures[88] = new Treasure("Magic crystal", 543);
+        treasures[89] = new Treasure("The crucible", 221);
+        treasures[90] = new Treasure("Root", 1);
+        treasures[91] = new Treasure("The mirror of eternity", 356);
+        treasures[92] = new Treasure("Crystal ball", 753);
+        treasures[93] = new Treasure("Herbs", 467);
+        treasures[94] = new Treasure("The Hand of the Prophet", 3235);
+        treasures[95] = new Treasure("Mysterious boots", 256);
+        treasures[96] = new Treasure("Boots", 433);
+        treasures[97] = new Treasure("Lute stockings", 235);
+        treasures[98] = new Treasure("Very simple boots", 676);
+        treasures[99] = new Treasure("Rivera's dancing shoes", 4343);
+        return treasures;
```

Comment:
Are you really comfortable creating 100 objects like this instead of some kind of randomizer? What if there were 1000 objects?